README.md

```diff
@@ -19,7 +19,7 @@ If you have a `crossProject`, the setting must be used only in the JS part:
 ```scala
 lazy val myCross = crossProject.
   ...
-  jsSettings.(
+  .jsSettings(
```

Comment:
While you're here, and since it's relevant, you should also remove the `.` at the end of `crossProject.`. Otherwise the fixed example still doesn't compile (`...` notwithstanding).

sbt used to recommend the trailing-dot style in their docs, but I see that now they use the leading-dot style instead (which IMO is better).